cmp/input/HoistInputModel.js

```diff
@@ -116,6 +116,7 @@ export class HoistInputModel extends HoistModel {
     @observable.ref internalValue = null;    // Cached internal value
     inputRef = createObservableRef();        // ref to internal <input> element, if any
     domRef = createObservableRef();          // ref to outermost element, or class Component.
+    isDirty = false;
```

Comment:
I'm not clear on the need for this new flag? Under which circumstances would `doCommitInternal()` be called before `noteValueChange`?

Reply:
when you tab through the form, without making any changes

Reply:
or click in and out of it